ca/ocsp.go

```diff
@@ -0,0 +1,125 @@
+package ca
```

Comment:
Given that you're creating a file named `ocsp.go`, I think that the `GenerateOCSP` function (which is the only function which calls `.enqueue`) should also be moved into this file.

Reply:
I'd like to propose doing this in a later refactoring, where `GenerateOCSP` is extracted entirely from `CertificateAuthorityImpl` and is implemented by its own object. What do you think? If you like the idea, I'll file an issue.

Reply:
Sounds good. There's already a separate gRPC service just for generating OCSP; having its logic split out into a separate object makes sense too.

Maybe add a TODO() referencing the bug you file to the top of GenerateOCSP in this change?